bin/docker

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+docker compose up css js web
```

Comment:
Could you configure your editor to append a newline to the end of files? Most editors do this by default as it's a UNIX convention.